admin/client/src/legacy/AddToCampaignForm.js

```diff
@@ -9,7 +9,7 @@ $.entwine('ss', function($){
             if (dialog.length) {
                 dialog.open();
             } else {
-                dialog = $('<div id="add-to-campaign__dialog" />');
+                dialog = $('<div id="add-to-campaign__dialog" class="add-to-campaign__dialog" />');
```

Comment:
You shouldn't use id's, we will be using the Airbnb css guides to get going and it also states in there don't use ids https://github.com/airbnb/css and also in BEM docs http://getbem.com/naming/